Susi/Custom Views/Message Cells/ImageCell.swift

```diff
@@ -20,36 +22,122 @@ class ImageCell: ChatMessageCell {
 
     let imageView: AnimatedImageView = {
         let imageView = AnimatedImageView()
+        imageView.image = ControllerConstants.Images.placeholder
+        imageView.contentMode = .scaleAspectFill
+        imageView.layer.borderWidth = 2.0
+        imageView.layer.borderColor = UIColor.white.cgColor
         imageView.layer.cornerRadius = 16
         imageView.layer.masksToBounds = true
+        imageView.isUserInteractionEnabled = true
         return imageView
     }()
 
+    lazy var thumbUpIcon: IconButton = {
+        let button = IconButton()
+        button.image = ControllerConstants.Images.thumbsUp
+        button.addTarget(self, action: #selector(sendFeedback(sender:)), for: .touchUpInside)
+        return button
+    }()
+
+    lazy var thumbDownIcon: IconButton = {
+        let button = IconButton()
+        button.image = ControllerConstants.Images.thumbsDown
+        button.addTarget(self, action: #selector(sendFeedback(sender:)), for: .touchUpInside)
+        return button
+    }()
+
     override func setupViews() {
         super.setupViews()
         setupView()
     }
 
     func addImageView() {
-        messageTextView.text = ""
         textBubbleView.addSubview(imageView)
-        textBubbleView.addConstraintsWithFormat(format: "H:|-8-[v0]-5-|", views: imageView)
-        textBubbleView.addConstraintsWithFormat(format: "V:|-4-[v0]-5-|", views: imageView)
+        imageView.frame = CGRect(x: 0, y: 0, width: 200, height: 200)
```

Comment:
Any reason for hard coding this? @imjog

Reply:
@chashmeetsingh I am displaying square image and then tapping on the image, full image will open in the browser. 200x200 was looking fine to me. Any changes?

Reply:
I guess this is the size of the image view inside the chat bubble? Should be in a ratio form like 3/5 or 1/2 (something similar). On older devices, the bubbles would not look good, I guess.

Reply:
@chashmeetsingh Okay, I am creating a new issue and sending a PR soon.